string Action
uint8 Target
---
bool isValid
